driver/command/command_nix_test.go

```diff
@@ -114,7 +114,7 @@ func TestCommandDriverOutputs(t *testing.T) {
 			},
 		}
 		_, err := cmddriver.Run(&op)
-		assert.Errorf(t, err, "Command driver (test-outputs-missing.sh) failed for item: /cnab/app/outputs/output2 no output value found and no default value set")
+		assert.NoError(t, err)
 	}
 	CreateAndRunTestCommandDriver(t, name, content, testfunc)
 	// Test for an output missing with default value present
```

Comment:
I wonder if we just update the comment here (maybe elsewhere) to clarify that now, even if a default for an output is declared in the bundle, it won't necessarily show up in the Operation Result's outputs list unless actually produced during the operation?

Reply:
It doesn't show up in the intermediate OperationResult returned by the driver but it is set by the runtime ultimately before the caller gets the OperationResult back from the Action.

So I'm not sure where/how to document that or if that's necessary? Ultimately I moved that burden of defaulting unproduced outputs off the drivers and onto the runtime. I think that may make more sense to anyone coming in fresh (and who didn't see the original patch we had in place to get output defaults working). Drivers really care about executing the requested operation and less about enforcing every single side requirement of the runtime spec, which output defaults fall into (and probably shouldn't have ever been a responsibility of each driver in the first place).

But if you see a spot to call out where it's set, we can add something. I think cnab-go could really benefit from a higher level doc that just explains the flow of what is going on. I started something like that with the go doc examples, but we could use more that just explain how we have implemented the runtime perhaps?